### ISSUE TITLE
Update to FontAwesome 4.6

### ISSUE THREAD

**Issue by blfpd** [plorcupine/latex-fontawesome#1]
This would be nice to have a newer version of FA! :blush:

**Comment by wusticality** [comment]
I agree with this, looks like the fonts have been split up now into separate files ..

**Comment by wusticality** [comment]
> This would be nice to have a newer version of FA! 😊

Just a heads up, I created a repo for this: https://github.com/wusticality/fontawesome-latex

**Comment by Loco169976** [comment]
😁
